Test/UnitTest1.cs

```diff
@@ -0,0 +1,73 @@
+using Training;
+
+namespace MSTestProject
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        MyList<int> myList = new ();
+
+        [TestMethod]
+        public void AddTest () {
+            myList.Add (1);
+            myList.Add (2);
+            List<int> list = new () { 1, 2 };
+            Assert.AreEqual (list.Count, myList.Count);
+            Assert.AreEqual (list.Capacity, myList.Capacity);
+        }
+
+        [TestMethod]
+        public void RemoveTest () {
+            myList.Add (1);
+            myList.Add (2);
+            List<int> list = new () { 1, 2 };
+            myList.Remove (1);
+            list.Remove (1);
+            Assert.ThrowsException<InvalidOperationException> ( () => myList.Remove(5));
+            Assert.AreEqual (list.Count, myList.Count);

```

Comment:
Since Remove (..) method returns a boolean, test should cover both the cases; when it returns _true_ and _false_.